lib/spack/docs/binary_caches.rst

```diff
@@ -164,7 +164,7 @@ List of popular build caches
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * `Extreme-scale Scientific Software Stack (E4S) <https://e4s-project.github.io/>`_: `build cache <https://oaciss.uoregon.edu/e4s/inventory.html>`_
-
+* `Spack Build Cache <https://cache.spack.io/>`_: `binary_mirror <https://binaries.spack.io/releases/v0.18>`_
```

Comment:
I get an error when I try to access `https://binaries.spack.io/releases/v0.18`. Would it be sufficient to just include `https://binaries.spack.io`?

Reply:
@tgamblin can probably best answer this. I took the binary mirror information from here: https://spack.io/spack-binary-packages/